python/truenas_storage_monitor/truenas_client.py

```
@@ -362,44 +363,50 @@ def get_volume_snapshots(self, volume_name: str) -> List[SnapshotInfo]:
+                                snap_data.get("properties", {}).get("used", {}).get("value", "0")
+                            ),
+                            referenced_size=int(
+                                snap_data.get("properties", {})
+                                .get("referenced", {})
+                                .get("value", "0")
+                            ),
                             full_name=snap_data.get("id", ""),
                         )
                         all_snapshots.append(snapshot)
             except Exception:
                 continue
```

Ruff (0.15.14)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)